### DISCUSSION TITLE
Inquiry for Account Recovery - Urgent

### DISCUSSION THREAD

**Discussion by guts-1469** [community/community#108499] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

My main account with the username "**amankr1469**" was suspended a week ago.

I didn't receive any prior warning or explanation about the suspension, which left me confused and concerned about the status of my account.

I raised a ticket for recovery but didn't receive any word yet.

If anyone faced the same issue in the past, what should I do?

**Comment by queenofcorgis** [comment, marked as answer]
Hi @aman-1469,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 
As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
